### PR TITLE
Roadmap PNG > html/css

### PR DESCRIPTION
**Pull request by maplesyrupsucker**
Converted the graphic from PNG to a more easy to edit chunk of markup and styles. Will allow us to use translated strings and gracefully handle roadmap localization without the need for edits to adobe xd assets.

### CODE REVIEW

File: _includes/roadmap/roadmap.html [hunk 1]
```diff
@@ -24,12 +24,155 @@ <h2>{% t 'headings.roadmap' %}</h2>
...
+                <img src="media-kit/3-bitcoin-cash-logo-ot-medium.png" alt="Bitcoin Cash roadmap">
+                <h3 class="rm-col-title">
+                    Roadmap
+                </h3>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-sm-12 col-md-4 to-animate rm-scaling">
+                <h3 class="rm-col-title">
+                    Scaling
```
Comment (BUYMERCIER) [inline _includes/roadmap/roadmap.html]: Maybe we could fetch the content from the i18n YAML files so it gets automatically translated here?

Comment (maplesyrupsucker) [reply _includes/roadmap/roadmap.html]: Yea that's the idea - i think we should put that in a separate PR tho so we can just get the updates @Mengerian requested in asap.

We'll need to take the strings from transifex and merge them into the .yml files. 

If you have time tho to tackle that within this PR maybe we can hold off a day or two. Thoughts?

File: _includes/roadmap/roadmap.html [hunk 2]
```diff
@@ -24,12 +24,155 @@ <h2>{% t 'headings.roadmap' %}</h2>
...
+                        <p>(blockchain pruning)</p>
+                        <ion-icon class="underway" name="cog"></ion-icon>
+                    </li>
+                    <li>
+                        <h4>Schnorr Signatures</h4>
+                        <p>(batched signature validation)</p>
+                        <ion-icon class="underway" name="cog"></ion-icon>
+                    </li>
+                    <li>
+                        <h4>Adjustable Block Cap ABC</h4>
```
Comment (Mengerian) [inline _includes/roadmap/roadmap.html]: This should read "Adaptive Block Size"

Comment (maplesyrupsucker) [reply _includes/roadmap/roadmap.html]: Thanks - latest commit resolves this

File: _includes/roadmap/roadmap.html (hunk 1 above)
Comment (BUYMERCIER) [reply _includes/roadmap/roadmap.html]: I'm on it

**Pull request merged**